zerver/lib/message.py

```diff
@@ -246,7 +247,14 @@ class ReactionDict(object):
     @staticmethod
     def build_dict_from_raw_db_row(row):
         # type: (Dict[str, Any]) -> Dict[str, Any]
-        return {'emoji_name': row.get('emoji_name'),
+        # Although we don't store the emoji name in the database, we still include
+        # the emoji_name in all the reaction dicts so that we don't break the api.
+        codepoint = row.get('codepoint')
+        is_realm_emoji = row.get('is_realm_emoji')
```

Comment:
Hmm, I think we should be just using `row['foo']`, not `row.get('foo')`, in this function, so that we assert correctly that these rows contain all the fields.  Can you fix that in a prerequisite commit?